DEPLOY.md

```diff
@@ -0,0 +1,23 @@
+## Preparation
```

Comment:
Should the below steps be wrapped in "Create PR to make the following changes:" ?

Reply:
Yeah could be, good idea.